source/Outfit.h

```diff
@@ -45,8 +45,10 @@ class Outfit : public Weapon {
 	// An "outfit" can be loaded from an "outfit" node or from a ship's
 	// "attributes" node.
 	void Load(const DataNode &node);
+	bool IsDefined() const;
```

Comment:
I thought it important to draw the distinction between "just defined" vs semantically valid. For outfits, they are one and the same. But for other classes, there can be additional constraints, such that even if the item is defined, it might not be valid.

Additionally, were the method named `IsValid`, it could be interpreted erroneously for outfits that are not typically read from a file, such as a ship's attributes collection.